<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-Q37MKND" ignored="0" name="OPENAPI" tests="21" failures="2" timestamp="07 五月 2019 08:22:47 GMT" time="6.95" errors="0">
  <testcase name="ChargingDataNo" time="0.462" classname="com.bitnei.apitest.testcases.openapi.ChargingData"/>
  <testcase name="ChargingDataNoFrom" time="0.079" classname="com.bitnei.apitest.testcases.openapi.ChargingData">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&amp;quot;errorMsg&amp;quot;:{&amp;quot;actualValue&amp;quot;:null,&amp;quot;expectedValue&amp;quot;:&amp;quot;VALID&amp;quot;,&amp;quot;currentValue is &amp;quot;:null},&amp;quot;data&amp;quot;:{&amp;quot;actualValue&amp;quot;:null,&amp;quot;expectedValue&amp;quot;:&amp;quot;参数 from  不符合规则：yyyy-MM-dd HH:mm:ss&amp;quot;,&amp;quot;currentValue is &amp;quot;:null}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"errorMsg":{"actualValue":null,"expectedValue":"VALID","currentValue is ":null},"data":{"actualValue":null,"expectedValue":"参数 from  不符合规则：yyyy-MM-dd HH:mm:ss","currentValue is ":null}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.openapi.ChargingData.ChargingDataNoFrom(ChargingData.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- ChargingDataNoFrom -->
  <testcase name="ChargingDataNoTo" time="0.067" classname="com.bitnei.apitest.testcases.openapi.ChargingData">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&amp;quot;errorMsg&amp;quot;:{&amp;quot;actualValue&amp;quot;:null,&amp;quot;expectedValue&amp;quot;:&amp;quot;VALID&amp;quot;,&amp;quot;currentValue is &amp;quot;:null},&amp;quot;data&amp;quot;:{&amp;quot;actualValue&amp;quot;:null,&amp;quot;expectedValue&amp;quot;:&amp;quot;参数 to  不符合规则：yyyy-MM-dd HH:mm:ss&amp;quot;,&amp;quot;currentValue is &amp;quot;:null}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"errorMsg":{"actualValue":null,"expectedValue":"VALID","currentValue is ":null},"data":{"actualValue":null,"expectedValue":"参数 to  不符合规则：yyyy-MM-dd HH:mm:ss","currentValue is ":null}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.openapi.ChargingData.ChargingDataNoTo(ChargingData.java:116)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- ChargingDataNoTo -->
  <testcase name="ChargingDataNormal" time="0.545" classname="com.bitnei.apitest.testcases.openapi.ChargingData"/>
  <testcase name="ChargingDataUpOneDay" time="0.084" classname="com.bitnei.apitest.testcases.openapi.ChargingData"/>
  <testcase name="ChargingDataWrongTime" time="0.053" classname="com.bitnei.apitest.testcases.openapi.ChargingData"/>
  <testcase name="EnergyConservationNormal" time="0.107" classname="com.bitnei.apitest.testcases.openapi.EnergyConservation"/>
  <testcase name="EnergyConservationUpScope" time="0.062" classname="com.bitnei.apitest.testcases.openapi.EnergyConservation"/>
  <testcase name="EnergyConservationWrongDateFormat" time="0.075" classname="com.bitnei.apitest.testcases.openapi.EnergyConservation"/>
  <testcase name="EnergyConservationWrongTime" time="0.066" classname="com.bitnei.apitest.testcases.openapi.EnergyConservation"/>
  <testcase name="EnergyConservationInvalidtoken" time="0.033" classname="com.bitnei.apitest.testcases.openapi.RunningData"/>
  <testcase name="EnergyConservationNolicensePlate" time="0.05" classname="com.bitnei.apitest.testcases.openapi.RunningData"/>
  <testcase name="EnergyConservationNormal" time="1.341" classname="com.bitnei.apitest.testcases.openapi.RunningData"/>
  <testcase name="EnergyConservationUpScope" time="0.056" classname="com.bitnei.apitest.testcases.openapi.RunningData"/>
  <testcase name="ChargingDataNormal" time="0.63" classname="com.bitnei.apitest.testcases.openapi.ChargingDataTest"/>
  <testcase name="ChargingDataNormal" time="0.064" classname="com.bitnei.apitest.testcases.openapi.ChargingDataTest"/>
  <testcase name="ChargingDataNormal" time="0.059" classname="com.bitnei.apitest.testcases.openapi.ChargingDataTest"/>
  <testcase name="ChargingDataNormal" time="0.051" classname="com.bitnei.apitest.testcases.openapi.ChargingDataTest"/>
  <testcase name="ChargingDataNormal" time="0.061" classname="com.bitnei.apitest.testcases.openapi.ChargingDataTest"/>
  <testcase name="ChargingDataNormal" time="0.101" classname="com.bitnei.apitest.testcases.openapi.ChargingDataTest"/>
  <testcase name="ChargingDataNormal" time="0.938" classname="com.bitnei.apitest.testcases.openapi.ChargingDataTest"/>
</testsuite> <!-- OPENAPI -->
